twilight-http/src/client/mod.rs

```diff
@@ -276,6 +278,14 @@ impl Client {
         InteractionClient::new(self, application_id)
     }
 
+    /// Create an interface for using application role connections.
+    pub const fn role_connections(
```

Comment:
This is unacceptable (not your fault), it's way to contrived. We need a RFC for how to handle routes requiring an `application_id`.

Reply:
That's fine! I'll use this for now in my code as I need something that works immediately. I'll be looking forward to the RFC and the proper implementation of this feature but it is unlikely I'll be taking part in the RFC process itself in the near future.